src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +14,9 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+        return (user != null && user.getPassword().equals(password)) ? true : false;
```

Comment:
Consider simplifying the code. You can directly return the result of the condition instead of using if/else or ternary operation to return true/false. For example:

// Instead of this
if (condition) {
return true;
} else {
return false;
}

// You can do this
return condition;
This makes the code cleaner and more straightforward.
```suggestion
        return user != null && user.getPassword().equals(password);
```

Reply:
fixed